README.md

```
@@ -57,15 +148,30 @@ The algorithm which computes this JsonPatch currently generates following operat
+#### Jackson 3.x API
+```java
+JsonNode target = Jackson3JsonPatch.apply(JsonNode patch, JsonNode source);
+```
 Given a `patch`, it apply it to `source` JSON and return a `target` JSON which can be ( JSON object or array or value ). This operation  performed on a clone of `source` JSON ( thus, the `source` JSON is unmodified and can be used further). 
 
 ## To turn off MOVE & COPY Operations
-```xml
-EnumSet<DiffFlags> flags = DiffFlags.dontNormalizeOpIntoMoveAndCopy().clone()
-JsonNode patch = JsonDiff.asJson(JsonNode source, JsonNode target, flags)
+
+#### Jackson 2.x API
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)